Line/src/main/java/com/applovin/mediation/adapters/LineMediationAdapter.java

```diff
@@ -140,6 +143,18 @@ public void initialize(final MaxAdapterInitializationParameters parameters, fina
     @Override
     public void onDestroy()
```

Comment:
`onDestroy()` will not be called unless the publisher explicitly calls `MaxInterstitialAd#destroy()` or `MaxRewardedAd#destroy()` - which most do not.

It is usually called when the "ad hidden" callback is fired from the mediate SDK - your SDK will need to be able to detect the Activity being dismissed and fire the "ad hidden" callback appropriately.